en/peripherals/rtk_gps_cuav_c-rtk.md

```diff
@@ -0,0 +1,22 @@
+# CUAV C-RTK
+
+The [CUAV C-RTK GPS receiver](http://doc.cuav.net/gps/c-rtk/en/) is a Real Time Kinematics module for mass market launched by CUAV. The complete RTK system can be composed of two C-RTK modules (one for aircraft and one for base station).Using RTK, PX4 can get its position with centimetre-level accuracy, which is much more accurate than can be provided by a normal GPS.
```

Comment:
Add space after full stop here: `station).Using`

Reply:
"The complete RTK system can be composed of two C-RTK modules (one for aircraft and one for base station)" - is it a normal RTK system? Ie can it also support multiple rovers for a single base station?

Reply:
Make "Real Time Kinematics" a link like this `[Real Time Kinematics](../advanced_features/rtk-gps.md)`